dojo/forms.py

```diff
@@ -3195,7 +3218,62 @@ def __init__(self, *args, **kwargs):
         self.group = kwargs.pop('group', None)
         super(ConfigurationPermissionsForm, self).__init__(*args, **kwargs)
 
-        self.permission_fields = get_configuration_permissions_fields()
+        if get_system_setting('enable_github'):
```

Comment:
You have to rebase the branch to the actual dev branch. Here you re-introduce an old version of some code.